l3kernel/l3file.dtx

```diff
@@ -3372,7 +3372,7 @@
       }
     \seq_concat:NNN \l_@@_tmp_seq \l_@@_tmp_seq \g_@@_record_seq
     \seq_remove_duplicates:N \l_@@_tmp_seq
-    #1 { LaTeX/kernel } { file-list }
+    #1 { LaTeX / kernel } { file-list }
       { \seq_map_function:NN \l_@@_tmp_seq \@@_list_aux:n }
```

Comment:
???

Reply:
I was tidying up the formatting to make search-and-replace easier. I've changed here to just `{ kernel }`  with `#1` here including the `__kernel_` prefix.

Reply:
my comment was not about the space but about not using \__kernel here :-)

Reply:
As I said elsewhere, there wasn't a `\__kernel_msg_` function for the `show` operation: I've fixed that.